src/YoutubeAtom.tsx

```diff
@@ -246,95 +266,50 @@ export const YoutubeAtom = ({
     const [hasUserLaunchedPlay, setHasUserLaunchedPlay] = useState<boolean>(
         false,
     );
+    const [player, setPlayer] = useState<YoutubePlayerType | undefined>();
 
-    const [isPlayerReady, setIsPlayerReady] = useState<boolean>(false);
-    const [player, setPlayer] = useState<YT.Player | null>(null);
-
-    const loadVideo = () => {
-        setPlayer(
-            new window.YT.Player(`${videoMeta.assetId}`, {
-                videoId: `${videoMeta.assetId}`,
-                events: {
-                    onReady: () => setIsPlayerReady(true),
-                    // Issue with setting events on Youtube object (refer to useEffect below)
-                    // https://stackoverflow.com/a/17078152
-                    // onPlayerStateChange,
-                },
-                playerVars: {
-                    autoplay: 1, // Enabling autoplay because the video loads when a user clicks so when the youtube iframe loads, we want the video to load
-                },
-            }),
-        );
-    };
-
+    // Use YouTubePlayer to init video
     useEffect(() => {
-        if (player && isPlayerReady) {
-            // Issue with setting events on Youtube object
-            // https://stackoverflow.com/a/17078152
-            // @ts-ignore
-            player.addEventListener('onStateChange', youtubeCallback);
-        }
-    }, [player, isPlayerReady]);
+        // if window is undefined it is because this logic is running on the server side
+        if (typeof window === 'undefined') return;
 
-    useEffect(() => {
-        if (player) {
-            // we have to hoist Youtube onStateChange callback as there is no way to remove using player.removeEventListener
-            youtubeCallback = (e: YT.PlayerEvent) =>
-                onPlayerStateChangeAnalytics({
-                    // ts-ignore is used because onStateChange has different listener to what is actually sent
-                    // @ts-ignore
-                    e,
-                    setHasUserLaunchedPlay,
-                    eventEmitters,
-                    player,
-                });
+        if (videoMeta && !player) {
+            setPlayer(YouTubePlayer(`youtube-video-${videoMeta.assetId}`));
```

Comment:
As above, I suspect this can just be inline. There _may_ be something around `useLayout` but now I'm just making random stream of consciousness statements